### PR TITLE
lint: add spell check

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

**What type of PR is this?**

To check spell in `make lint`.

/kind cleanup

### CODE REVIEW

**Comment by daixiang0** [comment]
related to #1637

File: Makefile [hunk 1]
```diff
@@ -294,3 +304,7 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
+# no-phony target
+$(MISSPELL):
+	$(call fetch_tool,github.com/client9/misspell/cmd/misspell,$(MISSPELL_VERSION))
```
Comment (kevin-wangzefeng) [inline Makefile:310]: Misspell is already included in golangci-lint, can we simply use golangci-lint with `-E misspell`?

Comment (daixiang0) [reply Makefile:310]: Yeah we can, but need to add config file to ignore specific word "mosquitto" since linter config can not be specified by cmd-line.

**Comment by daixiang0** [comment]
Fix #1637

**Pull request merged**